### PR TITLE
feat(route): Add LMU

### LINKED ISSUE #16319: LMU Job Portal
### Category

Application Updates

### Website URL

https://www.lmu.de/en/about-lmu/working-at-lmu/job-portal/academic-staff/

### Website description

Job vacancies for academic staff in LMU

### What content should be included?

new job posts/vacancies

### Additional description

Many thanks for your work!

### This is not a duplicated rss request

- [X] I have searched [existing issues](https://github.com/DIYgod/RSSHub/issues) and [pull requests](https://github.com/DIYgod/RSSHub/pulls) to ensure this rss proposal has not already been requested

### PR DESCRIPTION
**Pull request by StarDxxx**
## Involved Issue / 该 PR 相关 Issue

Close #16319

## Example for the Proposed Route(s) / 路由地址示例

```routes
/lmu/jobs
```

## New RSS Route Checklist / 新 RSS 路由检查表
  
- [x] New Route / 新的路由
  - [x] Follows [Script Standard](https://docs.rsshub.app/joinus/advanced/script-standard) / 跟随 [路由规范](https://docs.rsshub.app/zh/joinus/advanced/script-standard)
- [ ] Anti-bot or rate limit / 反爬/频率限制
  - [ ] If yes, do your code reflect this sign? / 如果有, 是否有对应的措施?
- [x] [Date and time](https://docs.rsshub.app/joinus/advanced/pub-date) / [日期和时间](https://docs.rsshub.app/zh/joinus/advanced/pub-date)
  - [x] Parsed / 可以解析
  - [x] Correct time zone / 时区正确
- [ ] New package added / 添加了新的包
- [ ] `Puppeteer`

### CODE REVIEW

File: lib/routes/lmu/jobs.ts [hunk 1]
```diff
@@ -0,0 +1,95 @@
+import { Route } from '@/types';
+import got from '@/utils/got';
+import { parseDate } from '@/utils/parse-date';
+
+const apiUrl = 'https://jobs.b-ite.com/api/v1/postings/search';
+
+// 辅助函数：根据 value 查找对应的 label
+function findLabel(value: string, options: Array<{ value: string; label: string }>): string {
+    const option = options.find((option) => option.value === value);
+    return option ? option.label : value; // 如果找不到匹配项，返回 value 本身
```
Comment (TonyRL) [inline lib/routes/lmu/jobs.ts:10]:
```suggestion
    return option?.label ?? value; // 如果找不到匹配项，返回 value 本身
```

File: lib/routes/lmu/jobs.ts [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+                <p><strong>Application deadline:</strong> ${job.endsOn}</p>
+                <p><strong>Job Details:</strong></p>
+                <p><strong>About us:</strong></p>
+                ${job.custom.das_sind_wir || ''}<br>
+                <p><strong>Your qualifications:</strong></p>
+                ${job.custom.das_sind_sie || ''}<br>
+                <p><strong>Benefits:</strong></p>
+                ${job.custom.das_ist_unser_angebot || ''}<br>
+                <p><strong>Contact:</strong> ${job.custom.kontakt || ''}</p>
+            `,
```
Comment (TonyRL) [inline lib/routes/lmu/jobs.ts:69]: Please use [`art-template`](https://docs.rsshub.app/joinus/advanced/script-standard#rendering-templates)

File: lib/routes/lmu/jobs.ts [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+export const route: Route = {
+    path: '/jobs',
+    name: 'LMU Job Openings',
+    url: 'lmu.de',
+    example: '/lmu/jobs',
+    maintainers: ['StarDxxx'],
+    radar: [
+        {
+            source: ['/en/about-lmu/working-at-lmu/job-portal/academic-staff/'],
```
Comment (TonyRL) [inline lib/routes/lmu/jobs.ts:89]:
```suggestion
            source: ['www.lmu.de/en/about-lmu/working-at-lmu/job-portal/academic-staff/'],
```

File: lib/routes/lmu/jobs.ts [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+    return {
+        title: 'LMU Academic Staff Job Openings',
+        link: 'https://www.lmu.de/en/about-lmu/working-at-lmu/job-portal/academic-staff/',
+        item: items,
+    };
+}
+
+export const route: Route = {
+    path: '/jobs',
+    name: 'LMU Job Openings',
```
Comment (TonyRL) [inline lib/routes/lmu/jobs.ts:83]:
```suggestion
    name: 'Job Openings',
```

File: lib/routes/lmu/jobs.ts [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+    });
+
+    return {
+        title: 'LMU Academic Staff Job Openings',
+        link: 'https://www.lmu.de/en/about-lmu/working-at-lmu/job-portal/academic-staff/',
+        item: items,
+    };
+}
+
+export const route: Route = {
```
Comment (TonyRL) [inline lib/routes/lmu/jobs.ts:81]: Missing feed `categories`

File: lib/routes/lmu/namespace.ts [hunk 6]
```diff
@@ -0,0 +1,17 @@
+import type { Namespace } from '@/types';
+
+export const namespace: Namespace = {
+    name: 'LMU',
```
Comment (TonyRL) [inline lib/routes/lmu/namespace.ts:4]:
```suggestion
    name: 'Ludwig Maximilian University of Munich',
```

File: lib/routes/lmu/jobs.ts (hunk 2 above)
Comment (StarDxxx) [reply lib/routes/lmu/jobs.ts:69]: Thank you for your suggestion! 🙏

I have implemented the changes and refactored the code to use `art-template` for rendering the job postings' descriptions. This should now align with the project's coding standards.

Thanks again for your review!

File: lib/routes/lmu/jobs.ts (hunk 5 above)
Comment (StarDxxx) [reply lib/routes/lmu/jobs.ts:81]: Thank you for your suggestion!🙏

I have added categories.

Thanks again for your review!

**Pull request merged**